lms/djangoapps/course_home_api/outline/v1/tests/test_views.py

```diff
@@ -151,27 +146,24 @@ def test_course_staff_can_see_non_user_specific_content_in_masquerade(self):
         assert response.data['course_blocks'] is not None
         assert response.data['handouts_html'] is not None
 
-    @override_experiment_waffle_flag(COURSE_HOME_MICROFRONTEND, active=True)
     @override_waffle_flag(COURSE_ENABLE_UNENROLLED_ACCESS_FLAG, active=True)
     def test_handouts(self):
         CourseEnrollment.enroll(self.user, self.course.id)
         self.store.create_item(self.user.id, self.course.id, 'course_info', 'handouts', fields={'data': '<p>Hi</p>'})
         assert self.client.get(self.url).data['handouts_html'] == '<p>Hi</p>'
 
-    @override_experiment_waffle_flag(COURSE_HOME_MICROFRONTEND, active=True)
     def test_get_unknown_course(self):
         url = reverse('course-home-outline-tab', args=['course-v1:unknown+course+2T2020'])
         response = self.client.get(url)
         assert response.status_code == 404
 
-    @override_experiment_waffle_flag(COURSE_HOME_MICROFRONTEND, active=False)
+    @override_waffle_flag(COURSE_HOME_USE_LEGACY_FRONTEND, active=True)
     @ddt.data(CourseMode.AUDIT, CourseMode.VERIFIED)
     def test_waffle_flag_disabled(self, enrollment_mode):
```

Comment:
Update test name to `test_waffle_flag_enabled` or `test_legacy_view_enabled` (I think I prefer the second, but I leave it all to your discretion)